### ISSUE TITLE
Fix failing travis tests on master

### ISSUE THREAD

**Issue by jchodera** [choderalab/openmmtools#358]
Travis tests of `master` are [failing](https://travis-ci.org/choderalab/openmmtools/jobs/397894069#L3386-L3415):
```
======================================================================
FAIL: Doctest: openmmtools.integrators.ThermostatedIntegrator
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test/lib/python2.7/doctest.py", line 2226, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for openmmtools.integrators.ThermostatedIntegrator
  File "/home/travis/miniconda/envs/test/lib/python2.7/site-packages/openmmtools-0.13.3-py2.7.egg/openmmtools/integrators.py", line 227, in ThermostatedIntegrator
----------------------------------------------------------------------
File "/home/travis/miniconda/envs/test/lib/python2.7/site-packages/openmmtools-0.13.3-py2.7.egg/openmmtools/integrators.py", line 283, in openmmtools.integrators.ThermostatedIntegrator
Failed example:
    integrator.getTemperature()
Expected:
    Traceback (most recent call last):
    ...
    AttributeError: type object 'object' has no attribute '__getattr__'
Got:
    Quantity(value=380.0, unit=kelvin)
======================================================================
FAIL: Test is_quantity_close method.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/miniconda/envs/test/lib/python2.7/site-packages/openmmtools-0.13.3-py2.7.egg/openmmtools/tests/test_utils.py", line 82, in test_is_quantity_close
    assert is_quantity_close(quantity1, quantity2) is test_result
AssertionError
```

**Comment by andrrizzi** [comment]
Weird, that test ran ok the last time I touched that code. It may be related to a new version of some package. I'll take a look.

**Comment by andrrizzi** [comment]
This was due to a problem caused by clashing `setup.py` and `conda` installation. @Lnaden figured it out and fixed it in #362.
